tests/posix/semaphore/prj.conf

```diff
@@ -1,4 +0,0 @@
-CONFIG_PTHREAD_IPC=y
-CONFIG_MAIN_THREAD_PRIORITY=2
```

Comment:
This `CONFIG_MAIN_THREAD_PRIORITY` from semaphore test is not included in the new unified prj.conf.. is it not necessary? Does the sema test behavior change in any way without it?

Reply:
@nniranjhana : With unified test suite, changing main thread priority would be misleading and also there is no need of changing the thread priority even for semaphores. The behavior is same for all tests with and without main thread priority 2.